site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -262,6 +262,9 @@ Whether a parent email is required.
 [field]#{base_field_name}.familyConfiguration.parentRegistrationEmailTemplateId# [type]#[UUID]# [since]#Available since 1.8.0#::
 The unique Id of the email template to use for parent registration.
 
+[field]#{base_field_name}.formConfiguration.adminUserFormId# [type]#[UUID]# [since]#Available since 1.20.0#::
+The unique Id of the form to use form adminstrator registration of users.
```

Comment:
" to use form adminstrator"
should be
" to use for adminstrator" 

no?

Reply:
Yep. Definitely. Too many `form`s